Apps/WebClient/src/ClientApp/src/components/timeline/entryCard/entrycard.vue

```diff
@@ -22,6 +27,17 @@ export default class EntrycardTimelineComponent extends Vue {
     @Prop({ default: true }) allowComment!: boolean;
     @Prop({ default: true }) showDetailsButton!: boolean;
 
+    @Getter("getEntryComments", { namespace: "comment" })
+    entryComments!: (entyId: string) => UserComment[];
+
+    private get comments(): UserComment[] {
+        return this.entryComments(this.entry.id) || [];
+    }
+
+    private get commentsCount(): number {
+        return this.comments !== undefined ? this.comments.length : 0;
```

Comment:
No need to check for undefined since it will always be a valid array

Reply:
code updated